packages/php-wasm/compile/php/phpwasm-emscripten-library.js

```diff
@@ -330,19 +353,25 @@ const LibraryExample = {
 			const ProcInfo = {
 				pid: cp.pid,
 				exited: false,
-				stdinFd: stdinFd,
-				stdinIsDevice: stdinFd in PHPWASM.input_devices,
-				stdoutChildFd: stdoutChildFd,
-				stdoutParentFd: stdoutParentFd,
-				stderrChildFd: stderrChildFd,
-				stderrParentFd: stderrParentFd,
+				stdinFd: std[0] ? std[0].child : null,
+				stdinIsDevice: std[0]
+					? std[0].child in PHPWASM.input_devices
+					: null,
+				stdoutChildFd: std[1] ? std[1].child : null,
```

Comment:
I don't expect this to work, since this file is processed using some outdated JS engine, but perhaps I'm wrong?

```suggestion
				stdoutChildFd: std[1]?.child,
```